opendevin/action/fileop.py

```diff
@@ -19,13 +24,28 @@ def resolve_path(file_path):
 
 @dataclass
 class FileReadAction(ExecutableAction):
+    """
+    Reads a file from a given path up to 100 lines.
+    Default lines 0:100
```

Comment:
Default here definitely needs to be the full file--some agents aren't aware of the start/end params